pgpg/cli.py

```diff
@@ -1,2 +1,42 @@
+import click
+
+from pgpg.core.cli import cmd_init
+from pgpg.core.cli import cmd_migrate
+from pgpg.core.cli import cmd_info
+from pgpg.core.cli import cmd_seal
+from pgpg.core.cli import cmd_open
+from pgpg.core.cli import cmd_sign
+from pgpg.core.cli import cmd_verify
+from pgpg.core.cli import cmd_set
+from pgpg.core.cli import cmd_gpg
```

Comment:
convert to single line import. 
`from pgpg.core.cli import (<insert functions here>)`